extensions/wikia/AssetsManager/tests/AssetsManagerTest.php

```diff
@@ -109,30 +129,43 @@ public function testCheckIfGroupForSkin( $skin, $skinRegisteredInGroup, $isSkinS
 			->method( 'isStrict' )
 			->will( $this->returnValue( $isSkinStrict) );
 
-		$assetsManagerConfigMock = $this->getMock( 'AssetsConfig', ['getGroupSkin'] );
+		$assetsManagerConfigMock = $this->getMockBuilder( AssetsConfig::class )
+			->setMethods( ['getGroupSkin'] )
+			->getMock();
+
 		$assetsManagerConfigMock
 			->expects( $this->any() )
 			->method( 'getGroupSkin' )
 			->will( $this->returnValue( $skinRegisteredInGroup ) );
 
-		// This needs to be called because AssetsConfig is a private singleton inside AssetsManager class
-		$this->mockClass( 'AssetsConfig', $assetsManagerConfigMock );
-		$assetsManagerMock = $this->getMock( 'AssetsManager', null, [], '', false );
+		/** @var PHPUnit_Framework_MockObject_MockObject|AssetsManager $assetsManagerMock */
+		$assetsManagerMock = $this->getMockBuilder( AssetsManager::class )
+			->disableOriginalConstructor()
+			->getMock();
+
+		$assetsConfigProperty = new ReflectionProperty( AssetsManager::class, 'mAssetsConfig' );
+		$assetsConfigProperty->setAccessible( true );
+		$assetsConfigProperty->setValue( $assetsManagerMock, $assetsManagerConfigMock );
 
 		$this->assertEquals( $assetsManagerMock->checkIfGroupForSkin( 'foo', $wikiaSkinMock ), $expectedValue );
 	}
 
 	/**
+	 * @covers AssetsManager::checkAssetUrlForSkin()
 	 * @dataProvider checkAssetUrlForSkinDataProvider
 	 */
 	public function testCheckAssetUrlForSkin( $url, $isSkinStrict, $expectedValue ) {
-		$skinMock = $this->mockClassWithMethods('WikiaSkin', [
+		/** @var WikiaSkin $skinMock */
+		$skinMock = $this->getMockWithMethods(WikiaSkin::class, [
 			'isStrict' => $isSkinStrict,
 			'checkIfGroupForSkin' => false, // "block" all assets when in strict mode
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)